packages/core/src/testing/MockLogger.ts

```diff
@@ -0,0 +1,16 @@
+import { AbstractBifoldLogger } from '../services/AbstractBifoldLogger'
+
+export class MockLogger extends AbstractBifoldLogger {
```

Comment:
I thought these generally went into `__mock__` so that when you import
`import { appLogger, createAppLogger } from '@utils/logger'`

You get the mock object not the real one?